.github/workflows/_build-and-test-locally.yml

```diff
@@ -292,12 +292,18 @@ jobs:
       fail-fast: false
       matrix:
         pg_version: ${{ fromJson(inputs.pg-versions) }}
+        no_default_lfc: [lfc]
+        include:
+          - pg_version: 16
+            no_default_lfc: nolfc
     steps:
       - uses: actions/checkout@v4
+        if: ${{ matrix.no_default_lfc == 'lfc' || inputs.build-type == 'release' }}
```

Comment:
If we need to have additional amount of if-s for steps, I'd move this parameter to upper lever (`build_and_test.yml`)

Reply:
done